python/ray/rllib/tuned_examples/halfcheetah-appo.yaml

```diff
@@ -0,0 +1,29 @@
+halfcheetah-appo:
+    env: HalfCheetah-v2
+    run: APPO
+    stop:
+        episode_reward_mean: 9800
+    config:
+        vtrace: True
+        gamma: 0.99
+        lambda: 0.95
+        sample_batch_size: 512
+        train_batch_size: 4096
+        num_workers: 16
+        num_gpus: 1
+        broadcast_interval: 1
+        max_sample_requests_in_flight_per_worker: 1
+        num_data_loader_buffers: 1
+        num_envs_per_worker: 32
+        minibatch_buffer_size: 16
+        num_sgd_iter: 32

```

Comment:
Can you add a test to regression_tests for IMPALA supporting Pendulum-v0?

Also, update the rllib-envs.rst to say that IMPALA supports continouus spaces now.